data/whitelist.d/misc_whitelist.txt

```diff
@@ -102,3 +102,6 @@ www.sfr.fr
 www.tripadvisor.fr
 www.uptodown.com
 www.winamax.fr
+ps.pndsn.com
```

Comment:
I found this: https://www.reddit.com/r/pelotoncycle/comments/byzh53/urls_that_need_to_be_whitelisted_by_adblock/

pndsn.com is listed there as recommended to be whitelisted. I think best to add `*.pndsn.com` instead.